inc/admin.php

```diff
@@ -1356,46 +1356,73 @@ public function get_layout_directories(){
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/mirror-widgets' ),
+				'anchor' => __( 'Mirror Widgets Addon', 'siteorigin-panels' ),
+			),
 			array(
-				'text' => __( 'Get a Lightbox Addon for SiteOrigin Widgets', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url( 'plugin/lightbox' )
+				'text' => __( 'Upload multiple image frames at once to Widgets Bundle Slider and Image Grid type widgets with %link%.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/multiple-media' ),
+				'anchor' => __( 'SiteOrigin Premium', 'siteorigin-panels' ),
 			),
 			array(
-				'text' => __( 'Hide rows and widgets on specific devices with the Toggle Visibility Addon', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url( 'plugin/toggle-visibility' )
+				'text' => __( 'Add parallax background images to your slider type widgets with %link%.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/parallax-sliders' ),
+				'anchor' => __( 'SiteOrigin Premium', 'siteorigin-panels' ),
 			),
 			array(
-				'text' => __( 'Get a Testimonials Addon for SiteOrigin Widgets', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url( 'plugin/testimonials' )
-			),			
+				'text' => __( 'Hide rows and widgets on specific devices with the %link%.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/toggle-visibility' ),
+				'anchor' => __( 'Toggle Visibility Addon', 'siteorigin-panels' ),
+			),
 			array(
-				'text' => __( 'Get a Tooltip Addon for SiteOrigin Widgets', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url( 'plugin/tooltip' )
+				'text' => __( 'Get a %link% with SiteOrigin Premium.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/tooltip' ),
+				'anchor' => __( 'Tooltip Addon', 'siteorigin-panels' ),
 			),
 			array(
-				'text' => __( 'Get a Webfont Selector Addon for SiteOrigin Widgets', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url( 'plugin/web-font-selector' )
-			),					
+				'text' => __( 'Use Google Fonts in SiteOrigin Widgets with the %link%.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/web-font-selector' ),
+				'anchor' => __( 'Webfont Selector Addon', 'siteorigin-panels' ),
+			),
 			array(
-				'text' => __( 'Get fast email support for Page Builder with SiteOrigin Premium', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url()
+				'text' => __( 'Get fast email support for Page Builder with %link%.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url(),
+				'anchor' => __( 'SiteOrigin Preimum', 'siteorigin-panels' ),
 			),
 		);
+		if ( class_exists( 'woocommerce' ) ) {
+			$links[] = array(
+				'text' => __( 'Get the %link%. Create custom templates for the Product, Archives, Shop, Cart, and Checkout pages.', 'siteorigin-panels' ),
```

Comment:
Also includes `Thank You` page.

Reply:
I had it in there but removed it later to try and shorten the tip a bit.